<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>481</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #8B0000;
  font-size: 16px;</string>
    </property>
    <property name="title">
     <string>Ajouter un Article</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  </string>
     </property>
     <property name="text">
      <string>Identifiant</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  
</string>
     </property>
     <property name="text">
      <string>Titre</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  </string>
     </property>
     <property name="text">
      <string>Date d'article</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  
</string>
     </property>
     <property name="text">
      <string>Auteur</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  
</string>
     </property>
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LeId">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="LeTitre">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="LeDescription">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="LeAuteur">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>220</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAjouter">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>370</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #8B0000	;
  border: none;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  border-radius : 16px;</string>
     </property>
     <property name="text">
      <string>Ajouter </string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>280</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>731</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>320</y>
      <width>351</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #8B0000;
  font-size: 16px;</string>
    </property>
    <property name="title">
     <string>Modifier Ou Supprimer un article</string>
    </property>
    <widget class="QLineEdit" name="LeIdMS">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  
</string>
     </property>
     <property name="text">
      <string>Identifiant</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonModifier">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #8B0000	;
  border: none;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 11px;
  margin: 4px 2px;
  border-radius : 16px;</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSupprimer">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #8B0000	;
  border: none;
  color: white;
  padding: 10px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 11px;
  margin: 4px 2px;
  border-radius : 16px;</string>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>320</y>
      <width>381</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #8B0000;
  font-size: 16px;</string>
    </property>
    <property name="title">
     <string>Rechercher ou Trier les articles</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  
</string>
     </property>
     <property name="text">
      <string>Rechercher par Id</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  
</string>
     </property>
     <property name="text">
      <string>Rechercher par titre</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LeIdRT">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="LeTitreRT">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #8B0000	;
  border: none;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 11px;
  margin: 4px 2px;
  border-radius : 16px;</string>
     </property>
     <property name="text">
      <string>Trier par id</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #8B0000	;
  border: none;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 11px;
  margin: 4px 2px;
  border-radius : 16px;</string>
     </property>
     <property name="text">
      <string>Trier par titre</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1088</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
